### DISCUSSION TITLE
请问“合集和视频列表（如直播回放）”是否支持批量下载 或 “视频下载”功能里的手动输入下载是否支持自动命名

### DISCUSSION THREAD

**Discussion by GH-CN** [the1812/Bilibili-Evolved#3596] [提问 / Q&A]
一。我想要只下载合集里视频的音轨，但是批量下载功能似乎暂时只支持分p视频。
二。所以想尝试通过手动输入的方式批量下载，但是这种方式生成的链接文件没有名字，只有扩展名
![image](https://user-images.githubusercontent.com/107448206/184553353-6a4ea7b3-0289-4724-8a7a-1731e498ac9e.png)
![image](https://user-images.githubusercontent.com/107448206/184553334-99ad4cc2-cf8b-4df0-8810-e331c85e52e6.png)
请问如何解决或者是否有替代方案呢，视频数量较多每个视频单独在页面下载太费劲了，感谢大家0 0
补充：尝试通过批量导出到IDM，但是文件名仍然难以使用
![image](https://user-images.githubusercontent.com/107448206/184554237-e8b08906-a97b-4761-9422-49da3b93f84a.png)

**Comment by Geeyun-JY3** [comment]
部分应该是重复了：#3240

**Reply by GH-CN** [reply]
感谢回复，3240之前看到了，但是因为没有回复，所以不确定有没有这个功能。
且为了更好的阐明目的，就尽可能详细的描述了。
感谢您呀0 0

**Reply by Geeyun-JY3** [reply]
#3240 打了 label 的：#1297

**Reply by GH-CN** [reply]
噗，看到了，不好意思，之前没注意

**Reply by Geeyun-JY3** [reply]
（我不用下载相关功能）搜了一下 issue 和  discussion，文件名的问题似乎和输出方式有关

**Reply by GH-CN** [reply]
> （我不用下载相关功能）搜了一下 issue 和 discussion，文件名的问题似乎和输出方式有关

感谢回复。根据手动下载里的描述似乎是可以批量命名的。但是看了下输出格式似乎只有MPV方式可以编辑输出命令，而MPV描述里面又说该输出方式不支持纯音轨输出。不知道是否是因为我没有正确理解批量命名的方法，就来问一下了。苦笑.jpg

**Reply by GH-CN** [reply]
> （我不用下载相关功能）搜了一下 issue 和 discussion，文件名的问题似乎和输出方式有关

补充一下，我才发现通用格式里可以编辑批量命名方式，修改一下就可以正常用了。谢谢！

**Comment by the1812** [comment]
是哪些视频啊, 我试了下手动输入也是有标题的啊

**Reply by GH-CN** [reply]
噗，您猜怎么着。我给您回复的时候，无意点开通用设置，才发现通用设置里可以编辑“批量命名格式”。
“批量命名格式”默认为“[n - ][ep]“，修改为”[title]“就正常获取了。
根据之前的测试，按照“批量命名格式”的默认参数，是可以获取多P视频标题的。无法获取单视频标题。可能是因为单视频没有”[ep]“参数嘛...?
感谢开发大大！

**Reply by the1812** [reply]
这个我记得在手动输入的说明里写了的，得包含 title 参数

**Reply by GH-CN** [reply]
> 这个我记得在手动输入的说明里写了的，得包含 title 参数

是的，对不起，我是笨蛋（哭。我知道需要改，但是我不知道在通用设置里
能再冒昧问下您，IDM导入ef2文件，下载文件名不是标题怎么办么。我看到ef2文件里有”filename“是正常的

**Reply by GH-CN** [reply]
> 这个我记得在手动输入的说明里写了的，得包含 title 参数

用Aria2正常全部下载了。非常感谢大大们！（比心

**Reply by the1812** [reply]
IDM 可以去 #2871 问问, 这个可能不是 IDM 默认就有的功能
